ansible/environments/mac/hosts

```diff
@@ -0,0 +1,33 @@
+; the first parameter in a host is the inventory_hostname which has to be
+; either an ip
+; or a resolvable hostname
+
+; used for local actions only
```

Comment:
Right - this file is the same as https://github.com/apache/incubator-openwhisk/blob/master/ansible/environments/local/hosts